src/core/electrostatics_magnetostatics/dipole.hpp

```diff
@@ -68,35 +68,6 @@ struct Dipole_parameters {
 extern Dipole_parameters dipole;
 
 namespace Dipole {
-/** Number of electrostatic contributions to the system pressure calculation. */
-constexpr size_t pressure_n() { return 0; }
-
-/** Number of electrostatic contributions to the system energy calculation.
- *  - slot 0: energies from particle pairs and magnetic field constraints
- *  - slot 1: energies from magnetostatics solvers
- *  - slot 2: energy corrections
```

Comment:
By removing this dependency, we also lose the attached documentation. Based on our offlline discussion last week, losing this docstring should not be a problem in the final state of the observable framework. Until then, we just have to make sure no one accidentally does the following:
```diff
--- a/src/core/Observable_stat.cpp
--- b/src/core/Observable_stat.cpp
@@ -31,1 +31,6 @@
+ #ifdef DIPOLES
  auto constexpr n_dipolar = 3;
+ #else
+ auto constexpr n_dipolar = 0;  // bug: a constraint actually writes to the
+                                // obs, even when DIPOLES is not compiled in
+ #endif
```

Reply:
I'm not sure I understand your point.

Reply:
These docstrings were introduced at the request of @RudolfWeeber in the last `Observable_stat` refactor. When doing the code review for this PR, I was about to propose adding feature guards on the `n_coulomb` and `n_dipolar` variables to be consistent with the feature guard on `n_vs`. By reading the docstrings you removed, I realized you didn't add feature guards on `n_dipolar` because of a side effect from a magnetostatics constraint.

Reply:
I see. To me this does not seem right, e.g. the type should not contain comments about how clients use it. Also wouldn't that just cause a compiler error exactly at the place of use, which is better information?

Reply:
right, the Spans do range checking. I was reasoning on the old implementation for some reason.